posix.mak

```diff
@@ -337,10 +337,10 @@ dlangspec.verbatim.txt : $(DMD) verbatim.ddoc dlangspec-consolidated.d
 # dmd compiler, latest released build and current build
 ################################################################################
 
-$(DMD):
+$(DMD) : ${DMD_DIR}/.cloned
 	${MAKE} --directory=${DMD_DIR}/src -f posix.mak -j 4
 
-$(DMD_REL):
+$(DMD_REL) : ${DMD_DIR}-${LATEST}/.cloned
```

Comment:
Please make those depend on `${DMD_DIR}/src/dmd` to address the main point of #1500.